### ISSUE TITLE
Annuli Crashes rack, cannot terminate process

### ISSUE THREAD

**Issue by Sequentonal** [gbrandt1/southpole-vcvrack#40]
Windows 7 and 10.
As stated in the header when I add Annuli to my patch, give it a couple of minuted and the rack crashes, audio keeps looping until I restart the PC. I can't terminate the Process through Process explorer.
I realize this is still being ported but i thought I'd mention it here. 
If there's any crash dump or further info that might be useful to ye i'd be happy to provide it.. just point me in the right direction :)
Really looking forward to having this great suite back in all it's glory :)

**Comment by SteveRussell33** [comment]
Try my beta build from here: https://github.com/SteveRussell33/southpole-vcvrack/releases
Annuli has had a small update - although it shouldn't impact any patch.
Let me know how it works out for you

**Comment by fractalgee** [comment]
It is at it again since yesterday, resurrecting an idea I had using 4 of them. Oddly enough now bails on loading any patch with Annuli in it, and no longer leaves a zombie faceless Rack process hanging about, so that part is an improvement.... Though not sure why that would be.  There is no improved v1 beta build as far as I can see on that github link @SteveRussell33 as latest there is 0.5.4....

**Comment by SteveRussell33** [comment]
Fixed. I've added my build from here: https://github.com/gbrandt1/southpole-vcvrack/issues/36#issuecomment-544230931

**Comment by fractalgee** [comment]
> Fixed. I've added my build from here: [#36 (comment)](https://github.com/gbrandt1/southpole-vcvrack/issues/36#issuecomment-544230931)

yep, found it over there, thanks. So far no further crashes and zombies, so that looks good :)
